src/Samples/TestEmbedded/Net5App/Net5App.csproj

```diff
@@ -3,12 +3,12 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <Platforms>x64;x86</Platforms>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net5.0-windows</TargetFramework>
     <CsWinRTWindowsMetadata>10.0.19041.0</CsWinRTWindowsMetadata>
   </PropertyGroup>
   
   <ItemGroup>
-    <PackageReference Include="Microsoft.Windows.CsWinRT" Version="1.3.6-prerelease.211004.13" IncludeAssets="build" ExcludeAssets="analyzers;contentFiles;lib;metadata;runtimes" />
+    <PackageReference Include="Microsoft.Windows.CsWinRT" Version="0.7.1-private.0" IncludeAssets="build" ExcludeAssets="analyzers;contentFiles;lib;metadata;runtimes" />
```

Comment:
version?

Reply:
I will update to one off the pipeline before merging, and then will update to the latest release when we drop embedded